### PR TITLE
Integration of the new dashboard loading animation

### PR DESCRIPTION
**Pull request by ferran98campos**
The commit replaces the 'Loading...' word for an animation of the armada ship sailing the sea. This animation will only appear while the users' dashboard is being loaded.

## Describe your changes
- Added a new image with the armada ship in #00d790 green
- Create the ship and wave animations
- The 'loading...' is replaced by that animation

Fixes: #1005

## Checklist before requesting review

- [x] Feature/fix PRs should add one atomic (as small as possible) feature or fix.
- [x] The code compiles and all the tests pass.

### CODE REVIEW

**Comment by ferran98campos** [comment]
Since the dashboard loading is relatively fast, here's an example of how the animation looks. 

[https://drive.google.com/file/d/1_ILMQ2aRRxxzH8WyipEkxJ6-w37CxBLz/view?usp=sharing](url)

File: apps/complete_registration/src/main.tsx [hunk 1]
```diff
@@ -73,8 +74,24 @@ export function IndexMainLogicWrapper() {
...
+                    </defs>
+                    <g className="la-parallax">
+                        <use xlinkHref="#la-ship" x="0" y="0"/>
+                        <use xlinkHref="#la-wave-1"/>
+                        <use xlinkHref="#la-wave-2"/>
+                        <use xlinkHref="#la-wave-3"/>
+                        <use xlinkHref="#la-wave-4"/>
+                    </g>
+                </svg>
+                <p className="font-medium pt-5">Loading...</p>
```
Comment (didrikmunther) [inline apps/complete_registration/src/main.tsx:94]: Could we extract this to a atomic component?

Comment (didrikmunther) [inline apps/complete_registration/src/main.tsx:94]: Could we extract this to a atomic component?

**Review by didrikmunther — CHANGES_REQUESTED** [review]
Looks great!

1. I left a comment
2. You need to run `pnpm build` and commit the build files because we haven't setup automatic building of this project yet unfortunately.

**Review by didrikmunther — APPROVED** [review]
LGTM, very nice!

**Review by Hampfh — APPROVED** [review]
Nice work!

**Pull request merged**